src/applications/mhv/secure-messaging/components/ThreadList/ThreadListItem.jsx

```diff
@@ -81,11 +81,14 @@ const ThreadListItem = props => {
       <div className="unread-column vads-l-col">
         {activeFolder.folderId !== DefaultFolders.DRAFTS.id &&
           (unreadMessages && (
-            <i
-              aria-hidden="true"
-              className="unread-icon vads-u-margin-right--1 vads-u-color--primary-darker fas fa-solid fa-circle"
-              data-testid="thread-list-unread-icon"
-            />
+            <span>
+              <i
+                className="unread-icon vads-u-margin-right--1 vads-u-color--primary-darker fas fa-solid fa-circle"
+                data-testid="thread-list-unread-icon"
+                alt="Unread message"
```

Comment:
Just flagging if `role="img"` needs to be added here as well as in instance above 🤷🏻‍♂️